From adb7b404674f8e2b5e9fbd566cebaed901167719 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 22 Dec 2022 16:05:08 +0000
Subject: [PATCH] GCC: fix incomplete type of gl::Presenter

GCC requires full definition of gl::Presenter for return type
of gl::GLOzoneEGL::CreateGLContext() with base::scoped_refptr.

Bug: 819294
Change-Id: I807beb8ef068f7ddb822a2e7f2cb48517ae62768
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4110925
Reviewed-by: ccameron chromium <ccameron@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Reviewed-by: Avi Drissman <avi@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1086362}
---
 ui/ozone/common/gl_ozone_egl.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/ozone/common/gl_ozone_egl.cc b/ui/ozone/common/gl_ozone_egl.cc
index 9b1d37e3a0415..e745dd71d3ad3 100644
--- a/ui/ozone/common/gl_ozone_egl.cc
+++ b/ui/ozone/common/gl_ozone_egl.cc
@@ -13,6 +13,7 @@
 #include "ui/gl/gl_share_group.h"
 #include "ui/gl/gl_surface.h"
 #include "ui/gl/gl_utils.h"
+#include "ui/gl/presenter.h"
 
 namespace ui {
 
-- 
2.39.1

